homeassistant/helpers/config_validation.py

```diff
@@ -1129,6 +1130,7 @@ def make_entity_service_schema(
                     vol.Remove("metadata"): dict,
                     **schema,
                     **ENTITY_SERVICE_FIELDS,
+                    vol.Optional(ATTR_INCLUDE_HIDDEN): vol.Boolean(),
```

Comment:
I haven't quite wrapped my head around how ENTITY_SERVICE_FIELDS gets used vs TARGET_SERVICE_FIELDS
and if I were to add my new field to those instead, do I move the "at least one of" validation up to those constants despite them probably intentionally being wrapped in vol.Schema.